### PR TITLE
Bug 1195992 - Create Imagecompare RTL test for Settings:Storage

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=50fcd37809292383cc9c73df75d7163e2e2c7def

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py [hunk 1]
```diff
@@ -123,6 +124,14 @@ def confirm_usb_storage(self):
...
         element.tap()
 
+    def cancel_usb_storage(self):
+        element = Wait(self.marionette).until(
+            expected.element_present(
+                *self._usb_storage_cancel_button_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+
+
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py]: One empty line too many

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_storage_RTL.py [hunk 2]
```diff
@@ -1,22 +1,55 @@
...
+            mediastorage_page.cancel_format_storage()
+            mediastorage_page.tap_select_media_location()
+            self.take_screenshot('media_storage-select_media_loc')
+            mediastorage_page.confirm_select_media_location()
+            self.take_screenshot('media_storage-media_locations')
+            mediastorage_page.pick_media_location('Internal')
+            settings.return_to_prev_menu(settings.screen_element)
+
+        ########### Application Storage #############################
+        if options == "disable":
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_storage_RTL.py]: I don't understand, why is this if all there? Basically this is a continuation of the 'if options == "disable":' thing, so no need to add this if call here.

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_storage_RTL.py [hunk 3]
```diff
@@ -1,22 +1,55 @@
...
+            settings.open_application_storage()
+            self.take_screenshot('application_storage')
+            settings.return_to_prev_menu(settings.screen_element)
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_storage_RTL.py]: Why have you written this line? You are not testing anything after it, so you could just stay on this page.

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_storage_RTL.py (hunk 2 above)
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_storage_RTL.py]: Or did you mean "enable" for this if? It doesn't seem like you test the "enable" option a lot in this test.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=03fc2836844a2c4b2ef9d7ade0b869f93e1b0e03

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3a92a9a88e49e1dfc3cf8d88c7df56c788989c4e

**Pull request merged**